jobs/rep_windows/spec

```diff
@@ -46,7 +46,10 @@ properties:
     default:
       - windows2012R2:/tmp/windows2012R2
   diego.rep.sidecar_rootfs_path:
-    description: "absolute_path representing the root filesystem used for sidecar processes (ie. '/tmp/windows2012R2'). Must be one of the preloaded_rootfses paths. Leaving the default empty string is ok, as it will then select the first of he preloaded_rootfses"
+    description: "absolute_path representing the root filesystem used for sidecar processes (ie. '/var/vcap/packages/cflinuxfs4/rootfs.tar'). Must be one of the preloaded_rootfses paths. If provided, this value will take precidence over diego.rep.sidecar_rootfs. Leaving the default empty string is ok, as it will then select the first of he preloaded_rootfses"
```

Comment:
```suggestion
    description: "absolute_path representing the root filesystem used for sidecar processes (ie. '/var/vcap/packages/cflinuxfs4/rootfs.tar'). Must be one of the preloaded_rootfses paths. If provided, this value will take precidence over diego.rep.sidecar_rootfs. If this property is left as the default empty string and diego.rep.sidecar_rootfs is left as the default empty string, then it will then select the first of the preloaded_rootfses. "
```